src/api/models/iuf/iuf-manifest-schema.yaml

```
@@ -399,19 +415,28 @@ properties:
                     default to the name of the directory containing the image
                     artifacts.
                   type: string
-                rootfs_file:
+                rootfs:
                   description: >
-                    The name of the file defining the root filesystem for the
-                    image. This file should be in squashfs format.
-                  type: string
-                kernel_file:
+                    The image root squashfs file for the image.
+                  $ref: '#/$defs/IMSArtifact'
+                kernel:
                   description: >
-                    The name of the kernel file for the image.
-                  type: string
-                initrd_file:
+                    The kernel file for the image.
+                  $ref: '#/$defs/IMSArtifact'
+                initrd:
                   description: >
-                    The name of the initrd file for the image.
-                  type: string
+                    The initrd file for the image.
+                  $ref: '#/$defs/IMSArtifact'
+          content_dirs:
```

A blank line before `content_dirs` would probably be helpful here for readability and consistency with the one added on line 391.

```suggestion

          content_dirs:
```